benchmarks/benchmarks/readwrite.py

```diff
@@ -0,0 +1,101 @@
+"""
+This module will benchmark io of Scanpy readwrite operations
+
+Things to test:
+
+* Read time, write time
+* Peak memory during io
+* File sizes
+
+Parameterized by:
+
+* What method is being used
+* What data is being included
+* Size of data being used
```

Comment:
Are any of these parametrized?

Reply:
Or is this just a wish-list?

Reply:
we just have the dataset “parametrized”, and we only specify one. I feel like that is more like an example of how parametrization looks like too. I could just simplify it to not use a parameter.